README.md

```diff
@@ -78,24 +49,15 @@ be seen in [cnf-certification-test-partner](https://github.com/test-network-func
 the first entry found with `"default"=true` is used. This annotation is automatically managed in OpenShift but may not
 be present in K8s.
 
-### Container/pod based Specs
+If a pod is not suitable for network connectivity tests because it lacks binaries (e.g. `ping`), it should be
+given the label `test-network-function.com/skip_connectivity_tests` to exclude it from those tests. The label value is
+not important, only its presence. Equivalent to `excludeContainersFromConnectivityTests` in the config file.
 
-* Pods to be tested by the `container` spec are identified with the `test-network-function.com/container=target`
-label. Any value is permitted but `target` is used here for consistency with the `generic` spec.
-* If specific tests are to be run on the pod then they can be listed as a JSON-encoded list of strings under the
-`test-network-function.com/container_tests` annotation. If the annotation is not found then by default every
-group of CNF tests defined in `testconfigure.yml` will be run.
+If label based discovery is not sufficient, this section can be manually populated as shown in the commented part of the [sample config](test-network-function/tnf_config.yml). However, instrusive tests need to be skipped ([see here](#disable-intrusive-tests)) for a reliable test result.
 
-For example:
-```yaml
-...
-  labels:
-    test-network-function.com/container: target
-  annotations:
-    test-network-function.com/container_tests: "[\"PRIVILEGED_POD\",\"PRIVILEGED_ROLE\"]" # optional
-...
-```
-### Operator Spec
+#### operators
```

Comment:
#### operators

##### Configuration via config file
See below an commented example of adding a single operator to test:

```shell script
testTarget:
  operators:
    - name: etcdoperator.v0.9.4 # name of the operator to test
      namespace: default # namespace of the operator to test
      subscriptionName: etcd # Subscription name of the operator
      autogenerate: false # TBD
      tests: # List of tests to run on the operator
        - OPERATOR_STATUS
```
        
##### Auto-discovery with labels

For auto-configuration, do the following:

* CSVs to be tested by the `operator` spec are identified with the `test-network-function-com/operator=target`
label. Any value is permitted but `target` is used here for consistency with the other specs.